lib/msal-node/src/cache/serializer/Serializer.ts

```diff
@@ -20,18 +20,22 @@ import {
     SerializedAppMetadataEntity,
 } from "./SerializerTypes.js";
 
+/**
+ * This class serializes cache entities to be saved into in-memory object types defined internally
+ * @internal
+ */
 export class Serializer {
     /**
      * serialize the JSON blob
-     * @param data
+     * @param data - JSON blob cache
```

Comment:
Nit: The desc above and adding in the same line with a separator, are they making a difference in typedocs how they show up?

Reply:
They are. A warning is generated when the delimiter isn't used.